adoc/chapters/programming_interface.adoc

```diff
@@ -14999,7 +14999,12 @@ application is compatible with a device unless:
   in <<sec:optional-kernel-features>>; or
 * It is decorated with a [code]#[[sycl::device_has()]]# {cpp} attribute that
   lists an aspect that is not supported by the device, as described in
-  <<sec:kernel.attributes>>.
+  <<sec:kernel.attributes>>; or
+* The translation unit containing the kernel was compiled in a compilation
+  environment that does not support the device.  Each implementation defines
```

Comment:
Minor comment, should we phrase this as `... that is not supported by the device` to align the phrasing with the statements above?

Reply:
That phrasing doesn't seem accurate, though.  A device doesn't support a compilation environment, it's the other way around.